packages/govern-subgraph/src/lib/MiniACL.ts

```diff
@@ -0,0 +1,71 @@
+import { Address, Bytes } from "@graphprotocol/graph-ts";
+import { Role as RoleEntity } from "../../generated/schema";
+
+export function frozenRoles(roles: string[], selector: Bytes): void {
+  roles
```

Comment:
I decided to handle the frozen of roles in this way I think is similar to what you did, let me know what you think.

Reply:
Way better than what I did!